Cyborg/VectorView.swift

```diff
@@ -368,3 +435,26 @@ final class ThemeableGradientLayer: CAGradientLayer {
     }
 
 }
+
+extension CGSize {
```

Comment:
Do we want to make this private because otherwise every app using Cyborg will gain access to them. Not saying that is a bad thing but we maybe would want to add some headerdoc if that was the case.

Reply:
They have to be to be testable. They're actually `internal` though, so only code in Cyborg can use them.

Reply:
Ah of course they automatically default to `internal`. My apologies.